docs/source/getting_started/getting_started.md

```diff
@@ -423,7 +390,269 @@ curl -X POST "$URL/users/u1234/personalized_documents" \
+    --data '{
+        "document": { "id": { "document_id": "12345", "sub_id": 0 },
+        "include_properties": true
+    }'
+```
+
+This pattern of replacing document id string with a snippet id struct is also used in all other places
+a snippet id can be used.
+
+Be aware that using `{ "id": "12345" }` represents the whole document but `{ "document_id": "12345", "sub_id": 0 }` represents the first snippet specifically and for some APIs this makes a difference
+for documents which have multiple embeddings as described in the sections below.
+
+## Ingestion
+
+There are two APIs with which you can ingest documents in a way where it can
+produce multiple snippets:
+
+- using the [`split` option with `snippet`](#automatic-document-splitting)
+- using the [`split` option with `file`](#file-upload)
+
+be aware that you need to use sufficient long text for it to be split.
+
+You could use the example from the [`split` option section](#automatic-document-splitting) or use
+your own document, something along ~1000 words will likely generate multiple splits.
+
+## Semantic Search
+
+As mentioned the search results are based on snippets not documents, as such a search
+can yield multiple different snippets of the same document.
+
+The semantic search API can also be used based on a specific snippet (and for systems
+which have documents with multiple embeddings that is recommended):
+
+```bash
+curl -X POST "$URL/semantic_search" \
+    --header "authorizationToken: $FRONTOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "document": { "id": { "document_id": "effaffe", "sub_id": 2 },
+        "include_properties": true
+    }'
+```
+
+This will return the snippets of which the semantic representation is closest to the
+semantic representation of the second snippet of the given document.
+
+```{note}
+Currently the whole document used for search is excluded from the search results.
+```
+
+For personalize semantic search with provided history snippet ids can also be used instead of
+document ids:
+
+```json
+"personalize": {
+    "exclude_seen": true,
+    "user": {
+         "history": [
+             {
+                 "id": { "document_id": "valid_doc_id1", "sub_id": 0 },
+                 "timestamp": "2000-05-14T20:22:50Z"
+             },
+             {
+                 "id": { "document_id": "valid_doc_id1", "sub_id": 1 },
+                 "timestamp": "2000-05-15T20:22:50Z"
+             }
+         ]
+    }
+}
+```
+
+When the history contains a interaction with a document which has multiple snippets it will be
+processed similar to registering an interaction which each snippet of that document.
+
+`exclude_seen` currently always refer to whole documents, i.e. in the example above even if there
+is a snippet with the id `{ "document_id": "valid_doc_id1", "sub_id": 30 }` it would exclud a the
+whole document `"valid_doc_id1"` no matter how many snippets it has.
```

Comment:
```suggestion
`exclude_seen` always refers to whole documents.
In the example above, even if there were a snippet with id `{ "document_id": "valid_doc_id1", "sub_id": 30 }` it would exclude the whole document `"valid_doc_id1"` no matter how many snippets it has.
```